extensions/wikia/ARecoveryEngine/ResourceLoaders/ResourceLoaderAdEngineSourcePointCSBootstrap.php

```diff
@@ -4,4 +4,10 @@ class ResourceLoaderAdEngineSourcePointCSBootstrap extends ResourceLoaderAdEngin
 	const CACHE_BUSTER = 18;
 	const SCRIPT_URL = 'https://api.sourcepoint.com/script/bootstrap?version=1';
 	const FALLBACK_SCRIPT_URL = __DIR__ . '/../js/SourcePoint/bootstrap_fallback.js';
+
+
+	protected function getData() // TODO: remove me, I'm hack because current version of SP has bugs
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)